code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -156,6 +156,7 @@
 		/obj/item/storage/backpack/industrial = 1,
 		/obj/item/clothing/suit/hooded/wintercoat/engineering/atmos = 3,
 		/obj/item/clothing/under/rank/engineering/atmospheric_technician = 3,
+		/obj/item/clothing/under/rank/engineering/atmospheric_technician/skirt = 3,
```

Comment:
bloat, if anything move this to some vendor or make it spawn less stuff overall
We already have too much stuff in locker anyways and I feel like this should be niche option or just be in vendor for free

Reply:
This wardrobe literally doesn't spawn anywhere btw.

Reply:
And the engineering one only appears on Box in one instance, I'd say it's a unique map feature instead.

Reply:
Moved to a compression box following the convo in Discord.